examples/twobee/twobee/app.py

```diff
@@ -0,0 +1,38 @@
+import random
+import toga
+from toga.style.pack import Pack, COLUMN, CENTER
+
+HEADS = toga.Image('https://beeware.org/static/images/brutus-270.png')
+TAILS = toga.Image('https://beeware.org/project/projects/libraries/toga/toga.png')
+TOSS = toga.Image('https://beeware.org/project/projects/tools/cricket/cricket.png')
+IMG_SIZE = toga.style.Pack(height=75, width=75)  # TODO be nice if didn't have to specify size
+
+
+def build(_):
```

Comment:
While the "just a method" version of building an app is legal, I've been contemplating if we should deprecate it; while it's nice to have "an app is just a function" as an option, it stops being a useful abstraction so quickly that I wonder if it's actually useful pedagogically. Even in this example, the application state and the button on_press handlers should arguably be methods on the app, rather than a nested function definition.

Reply:
I think this would be a shame. OO apis I feel are less popular these days and in this case I don't think it adds a lot except extra indenting and additional syntax. In more complex apps nothing stops me creating an object to store my state that isn't subclassing App  and pass the constructor to the startup method argument of the App constructor. I think supporting both is friendly to different developer styles.